uk.ac.kcl.language.fsql/src/uk/ac/kcl/language/fsql/generator/FSQLGenerator.xtend

```diff
@@ -329,8 +329,8 @@ class FSQLGenerator extends AbstractGenerator {
 	 dispatch def String generateQuerySelect(GroupSortedSelect select, String table, WhereClause whereClause)'''
 	 «select.select.generateQuerySelect(table, whereClause)» «select.groupBy.generateGroupBy» «select.orderBy.generateOrderBy»;'''
 	 
-//	 dispatch def String generateQuerySelect(AggregatedSelect select, String table, WhereClause whereClause)'''
-//	 SELECT «select.aggregation.generateAggregation»(«select.select.column.generateSQLColumnNameReference») FROM «table» «whereClause.generateWhereClause»'''
+	 dispatch def String generateQuerySelect(AggregatedSelect select, String table, WhereClause whereClause)'''
+	 SELECT «select.aggregation.generateAggregation»( «select.select.column.generateSQLColumnNameReference» ) FROM «table» «whereClause.generateWhereClause»'''
 	 
```

Comment:
This is the final version. Is it ok @TeoMoisi ?

Reply:
Yes, I approved it.

Reply:
ship it